Volume/Views/Onboarding/PublicationDetail/PublicationDetail.swift

```diff
@@ -59,27 +59,37 @@ struct PublicationDetail: View {
     private var backgroundImage: some View {
         ZStack {
             GeometryReader { geometry in
+                let scrollOffset = geometry.frame(in: .global).minY
+                let headerOffset = scrollOffset > 0 ? -scrollOffset : 0
+                let headerHeight = scrollOffset > 0 ? geometry.size.height + scrollOffset : geometry.size.height
```

Comment:
An equally valid solution for clamping:
```suggestion
                let headerHeight = geometry.size.height + max(scrollOffset, 0)
```

Reply:
Love this math.